### PR TITLE
Added the !technicalSupport and !ideaPitch commands

### LINKED ISSUE #211: [Feature Request] - Joining queue to pitch your idea
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Is your feature request related to a problem, or an open issue? Describe the problem, and if applicable, reference the issue.

We want a space where people can give suggestions or ideas to better the Discord server.

## Describe the solution you'd like

Create a Queue bot to add suggestions.

## Describe alternatives you've considered

Here is a link to create a queue bot : [https://top.gg/bot/679018301543677959](https://top.gg/bot/679018301543677959)

## Additional context

Related to #212

### LINKED ISSUE #212: [Feature Request] - Technical Help in Discord
### Pre-requisites:
- [X] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Is your feature request related to a problem, or an open issue? Describe the problem, and if applicable, reference the issue.

Create the functionality in Discord that enables users to join to the Tech Support Queue. This functionality already exists on Slack bot Home screen as interactive buttons. We need to find a method of implementation to allow users to join to the support queue since Discord bot doesn't have a home screen. 

## Describe the solution you'd like

If the command is tagged "!getHelp" in any of the channels bot should be responding with the confirmation text and add the user to the support queue. 

## Describe alternatives you've considered

Support channel can be added to the Discord server to implement this same functionality. 

## Additional context

We will work with @TheDivexz on this issue. 

Related to #211

### PR DESCRIPTION
**Pull request by Ulfingrad**
implementing proper tests

## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Worked with @smitshah2531 to implement both the technical support and idea pitch queue commands to Discord. We had difficulty writing the tests for this section, so that will have to be implemented soon.

## Related Issues

Closes #211 as well as closes #212

### CODE REVIEW

**Comment by Ulfingrad** [comment]
Still no tests, but it should now but up to date with main.

File: src/discord/events/message/index.ts [hunk 1]
```diff
@@ -19,6 +20,18 @@ export const commands: Command[] = [
     description: 'Replies with pong',
     handler: ping,
   },
+  {
+    handlerId: 'ideaPitch',
+    trigger: '!ideaPitch',
+    description: 'Adds the user to the idea pitch queue',
```
Comment (SpencerKaiser) [inline src/discord/events/message/index.ts:26]:
```suggestion
    description: 'Think you have a good idea? Join a queue to come pitch to our team!',
```

File: src/discord/events/message/index.ts [hunk 2]
```diff
@@ -19,6 +20,18 @@ export const commands: Command[] = [
...
+    handler: supportRequest,
+  },
+  {
+    handlerId: 'techincalSupport',
+    trigger: '!technicalSupport',
+    description: 'Adds the user to the technical support queue',
```
Comment (SpencerKaiser) [inline src/discord/events/message/index.ts:32]:
```suggestion
    description: 'Need help with your hack? Join our tech support queue so our team can help!',
```

File: src/tests/api/supportRequest.test.ts [hunk 3]
```diff
@@ -48,7 +48,6 @@ describe('api/supportRequest', () => {
         Authorization: adminSecret,
       })
       .expect(200);
-
```
Comment (SpencerKaiser) [inline src/tests/api/supportRequest.test.ts:51]: Revert this change

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
Just a few quick changes and we're good to merge!

**Review by SpencerKaiser — APPROVED** [review]

**Pull request merged**